package.json

```diff
@@ -50,5 +50,394 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "acorn": "^7.4.1",
+    "acorn-jsx": "^5.3.1",
+    "aggregate-error": "^3.1.0",
+    "ajv": "^6.12.6",
+    "ansi-colors": "^4.1.1",
+    "ansi-escapes": "^4.3.2",
+    "ansi-regex": "^5.0.1",
+    "ansi-styles": "^4.3.0",
+    "argparse": "^1.0.10",
+    "array-includes": "^3.1.3",
+    "array-union": "^2.1.0",
+    "array.prototype.flat": "^1.2.4",
+    "array.prototype.flatmap": "^1.2.4",
+    "arrify": "^1.0.1",
+    "asn1": "^0.2.4",
+    "assert-plus": "^1.0.0",
+    "astral-regex": "^2.0.0",
+    "async": "^3.2.0",
+    "asynckit": "^0.4.0",
+    "autoprefixer": "^9.8.6",
+    "aws-sign2": "^0.7.0",
```

Comment:
That's a _lot_ of dependencies for just installing `jquery`. Also, this `package.json` is in the root of the repo, not in the `npm-intro` directory.